build.gradle.kts

```diff
@@ -4,6 +4,11 @@
  * Copyright 2016-Present Datadog, Inc.
  */
 
+plugins {
+    `maven-publish`
```

Comment:
why do we apply `maven-publish` here if we already have it applied at the sub-project level?

Reply:
Because it's required by the nexus plugin